02-technical-guides/hierarchies/02-hierarchy-manipulations.ipynb

```diff
@@ -0,0 +1,2029 @@
+{
```

Comment:
<p>Change this: </p><p> </p><p>Slicing hierarchies are used when we want a hierarchy to no longer be aggregable at the top level/across all members of a hierarchy. </p><p> </p><p>To =&gt;</p><p>The values associated with a hierarchy are by default aggregated to the top level, across all members of that hierarchy.  This makes sense and provides useful analysis for data that should be summed in this way.  But for some hierarchies, it makes more sense to specify one of the hierarchy members as a default or slicing member.  This is very common for dates, so that today's date is displayed instead of the aggregate of all the values since the beginning of time</p><p>(you'll have to reword that and paraphrase it a bit)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/atoti/pull/890/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Reply:
<p>After speaking with @printhellohetal, I've re-designed the <code>Using slicing hierarchies</code> section in general. This now reads as the following, with a corresponding code walkthrough:</p><p> </p><p><strong>Slicing hierarchies are used when we want a hierarchy that is no longer aggregable at the top level/across all of its underlying members. For example, within our data model imagine that for every order made, a corresponding record of inventory with an&nbsp;</strong><code>AsOfDate</code><strong>&nbsp;is taken for each product sold. We can assume that each product started at&nbsp;</strong><code>10,000</code><strong>&nbsp;items in inventory. In this case, it would not provide any meaningful information to aggregate the current state or quantity of inventory across all the&nbsp;</strong><code>AsOfDate</code><strong>&nbsp;member dates, nor would it be a good use of computational resources.</strong></p><p> </p><p><strong>Let's see how this works in practice.</strong></p>